src/omnisharp/launcher.ts

```diff
@@ -339,10 +339,16 @@ function getConfigurationValue(globalConfig: vscode.WorkspaceConfiguration, csha
 }
 
 async function launchDotnet(launchInfo: LaunchInfo, cwd: string, args: string[], platformInfo: PlatformInformation, options: Options, dotnetResolver: IHostExecutableResolver): Promise<LaunchResult> {
+    function escapeIfNeeded(arg: string) {
+        const hasSpaceWithoutQuotes = /^[^"].* .*[^"]/;
```

Comment:
What does this regex actually mean? A space not surrounded by quotes?

Reply:
yeah stole it from the launchWindows method. Could just check startsWith('"') and contains(' ') instead.

Reply:
I am slightly worried about quotes in the middle of arguments: should I be? Or will that break things elsewhere?

Reply:
> Could just check startsWith('"')

Make sure you account for leading/trailing spaces that don't matter to the command.